scripts/send-slack-notification-bot.py

```diff
@@ -61,20 +63,31 @@ def main():
         logging.error("❌ Error: Could not read version from cluster_version file.")
         sys.exit(1)
 
-    majorMinor = version.rsplit('.', 1)[0]
+    # Determine RHEL version based on the cluster version
+    rhel_version = "rhel9"  # Default to rhel9
+    try:
+        # Compare version as a tuple of integers, e.g., (4, 14) <= (4, 15)
+        major, minor, *_ = map(int, version.split('.'))
+        if (major, minor) <= (4, 15):
```

Comment:
if you have here RC version it may not work 
it will fail to check STR <= INT
```python
if (4, "18-rc0") <= (4, 15):
```

this is one more reason why you should use the `version` object 
it also make it more readable

Reply:
agreed offline to approve it - there are no  rc / ec releases been tested